run_analysis.py

```diff
@@ -1,109 +1,133 @@
-#from mpas_analysis.ocean.moc_timeseries import moc_timeseries
-from mpas_analysis.ocean.sst_modelvsobs import sst_modelvsobs
-
-from mpas_analysis.sea_ice.timeseries import seaice_timeseries
-from mpas_analysis.sea_ice.modelvsobs import seaice_modelvsobs
-
-
-#GENERATE OCEAN DIAGNOSTICS
-if config.getboolean('ohc_timeseries','generate'):
-    print ""
-    print "Plotting OHC time series..."
-    ohc_timeseries(config)
-
-if config.getboolean('sst_timeseries','generate'):
-    print ""
-    print "Plotting SST time series..."
-    sst_timeseries(config)
-
-if config.getboolean('nino34_timeseries','generate'):
-    print ""
-    print "Plotting Nino3.4 time series..."
-    #nino34_timeseries(config)
-
-if config.getboolean('mht_timeseries','generate'):
-    print ""
-    print "Plotting Meridional Heat Transport (MHT)..."
-    #mht_timeseries(config)
-
-
-if config.getboolean('moc_timeseries','generate'):
-    print ""
-    print "Plotting Meridional Overturning Circulation (MOC)..."
-    #moc_timeseries(config)
-    
-if config.getboolean('sst_modelvsobs','generate'):
-    print ""
-    print "Plotting 2-d maps of SST climatologies..."
-    sst_modelvsobs(config)
-
-
-#GENERATE SEA-ICE DIAGNOSTICS
-if config.getboolean('seaice_timeseries','generate'):
-    print ""
-    print "Plotting sea-ice area and volume time series..."
-    seaice_timeseries(config)
-
-if config.getboolean('seaice_modelvsobs','generate'):
-    print ""
-    print "Plotting 2-d maps of sea-ice concentration and thickness climatologies..."
-    seaice_modelvsobs(config)
-
-
-#GENERATE LAND-ICE DIAGNOSTICS
-
-
-if config.getboolean('plot','displayToScreen'):
-   plt.show()
+def path_existence(config, section, option, warn=False, ignorestr=None): #{{{
+    inpath = config.get(section, option)
+    if not (os.path.isdir(inpath) or os.path.isfile(inpath)):
+        # assumes that path locations of ignorestr won't return an error, e.g.,
+        # ignorestr="none" is a key word to indicate the path or file is
+        # optional and is not needed
+        if inpath == ignorestr:
+            return False
+        errmsg = "Path %s not found. Exiting..." % inpath
+        if warn:
+            print 'WARNING!!! ' + errmsg
+            return False
+        else:
+            raise SystemExit(errmsg)
+    return inpath #}}}
+
+def analysis(config): #{{{
+
+    # Checks on directory/files existence:
+    indir = path_existence(config, 'paths', 'archive_dir_ocn')
+
+    if config.get('case', 'ref_casename_v0') is not 'None':
+        indir_v0data = path_existence(config, 'paths','ref_archive_v0_ocndir')
+        indir_v0data = path_existence(config, 'paths','ref_archive_v0_seaicedir')
+
+    usesi = config.getboolean('seaice_timeseries', 'generate')
+    sicompobs = config.getboolean('seaice_timeseries', 'compare_with_obs')
+    simodelvobs = config.getboolean('seaice_modelvsobs', 'generate')
+    if (usesi and sicompobs) or simodelvobs:
```

Comment:
PEP8 standards are that names with underscores are for functions, but surprisingly it doesn't clearly specify variable names.  Also, typically variable names should be 5-10 chars long, hence my choice of variable names.  I was trying to avoid aliasing with known usage of functions but you are right, it is better in this case and I'll make this change.